src/CQRS/LeanCode.CQRS.AspNetCore/ServiceCollectionCQRSExtensions.cs

```diff
@@ -55,4 +74,10 @@ public CQRSServicesBuilder AddCQRSObjects(TypesCatalog contractsCatalog, TypesCa
         objectsSource.AddCQRSObjects(contractsCatalog, handlersCatalog);
         return this;
     }
+
+    public CQRSServicesBuilder AddCQRSObjects(IEnumerable<TypeInfo> possibleContracts, IEnumerable<TypeInfo> possibleHandlers)
```

Comment:
What does `possible` prefix means here and in the other place?

Reply:
It mean that if someone passes type that cannot be registered as CQRS object for any reason.

But now it's outdated anyway.